docs/guides/relay.md

```diff
@@ -0,0 +1,381 @@
+    first: Int = null
+    last: Int = null
+  ): FruitConnection!
+}
+```
+
+With only that we have a way to query `node` to retrieve any `Node` implemented
+type in our schema (which includes our `Fruit` type), and also a way to retrieve
+a list of fruits with pagination.
+
+For example, to retrieve a single fruit given its unique ID:
+
+```graphql
+query {
+  node(id: "<some id>") {
+    id
+    ... on Fruit {
+      name
+      weight
+    }
+  }
+}
+```
+
+Or to retrieve the first 10 fruits available:
+
+```graphql
+query {
+  fruitConnection(first: 10) {
+    pageInfo {
+      firstCursor
+      endCursor
+      hasNextPage
+      hasPreviousPage
+    }
+    edges {
+      # node here is the Fruit type
+      node {
+        id
+        name
+        weight
+      }
+    }
+  }
+}
+```
+
+### Custom connection pagination
+
+The default `Connection` implementation uses a limit/offset approach to paginate
+the results. This is a basic approach and might be enough for most use cases.
+
+<Note>
+
+`Connection` implementes the limit/offset by using slices. That means that you can
+override what the slice does by customizing the `__getitem__` method of the object
+returned by `resolve_nodes`.
+
+For example, when working with `Django`, `resolve_nodes` can return a `QuerySet`,
+meaning that the slice on it will translate to a `LIMIT`/`OFFSET` in the SQL
+query, which will fetch only the data that is needed from the database.
+
+Also note that if that object doesn't have a `__getitem__` attribute, it will
+use `itertools.islice` to paginate it, meaning that when a generator is being
+resolved it will only generate as much results as needed for the given pagination,
+the worst case scenario being the last results needing to be returned.
+
+</Note>
+
+You may want to use a different approach to paginate your results. For example,
+a cursor-based approach. For that you need to subclass the `Connection` type
+and implement your own `from_nodes` method. For example, suppose that in our
+exaple above, we want to use the fruit's weight as the cursor, we can implement
+it like that:
+
+```python
+from strawberry.relay import to_base64
+
+
+@strawberry.type
+class FruitCustomPaginationConnection(strawberry.relay.Connection[Fruit]):
+    @classmethod
+    def from_nodes(
+        cls,
+        nodes: Iterable[Fruit],
+        *,
+        info: Optional[Info] = None,
+        total_count: Optional[int] = None,
+        before: Optional[str] = None,
+        after: Optional[str] = None,
+        first: Optional[int] = None,
+        last: Optional[int] = None,
+    ):
+        # Note that this is a showcase implementation and is far from
+        # being optimal performance wise
+        edges_mapping = {
+            to_base64("fruit_name", n.name): strawberry.relay.Edge(
+                node=n,
+                cursor=to_base64("fruit_name", n.name),
+            )
+            for n in sorted(nodes, key=lambda f: f.name)
+        }
```

Comment:
this is the best place for allowing to customise edges?

Reply:
Currently, yes